README.md

```diff
@@ -115,7 +146,7 @@ Building without a container requires a bit more manual work and setup in your e
 
 Install the necessary dependencies:
 ```sh
-Fedora$ sudo dnf install gpgme-devel libassuan-devel btrfs-progs-devel device-mapper-devel
+Fedora$ sudo dnf builddep skopeo
```

Comment:
See #412, this may not be correct.

Reply:
And I thought the #412 change was a nice shortcut.  But as you noted, if they don't always match we should list them out.  I've reverted this and have added ostree-devel as you noted.